code/script.js

```diff
@@ -1,9 +1,30 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+const toggleActiveClass = () => {
+  document.getElementById('q1').classList.toggle('active')
+  document.getElementById('a1').classList.toggle('active')
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+const toggleActiveClassSecond = () => {
+  document.getElementById('q2').classList.toggle('active')
+  document.getElementById('a2').classList.toggle('active')
+}
+
+const toggleActiveClassThird = () => {
+  document.getElementById('q3').classList.toggle('active')
+  document.getElementById('a3').classList.toggle('active')
+}
+
+const toggleActiveClassFourth = () => {
+  document.getElementById('q4').classList.toggle('active')
+  document.getElementById('a4').classList.toggle('active')
+}
+
+const toggleActiveClassFifth = () => {
+  document.getElementById('q5').classList.toggle('active')
+  document.getElementById('a5').classList.toggle('active')
+}
```

Comment:
You could actually narrow the functions down to one that can take all q1-q5. So if you want a general function it can be called for example
```
const toggle = () => {
}
```
Inside your function now you have `document.getElementById('q5').classList.toggle('active')`. We can make that with out general function as well. But we need to pass on `q5` so we know what element we should toggle `active` on.
```
const toggle = (questionId) => {
  console.log(questionId); //output: 5
}

document.getElementById('q5').onclick = toggle(5);
```
We can then use our questionId in your code
```
const toggle = (questionId) => {
  document.getElementById('q' + questionId).classList.toggle('active')
}

document.getElementById('q5').onclick = toggle(5);
```
That's all you need! What you can do next is to add the `a5` as well :)